test/exe/main_common_test.cc

```diff
@@ -104,6 +104,31 @@ TEST_P(MainCommonTest, ConstructDestructHotRestartDisabledNoInit) {
   EXPECT_TRUE(main_common.run());
 }
 
+// This test verifies that validation can run from MainCommonBase, from a thread
+// other than the test thread. This is a desirable calling sequence in some
+// contexts. To make this work we must declare MainThread in MainCommonBase, in
+// addition to declaring it in MainCommon. There is no harm in double-declaring.
+TEST_P(MainCommonTest, ValidateUsingMainCommonBaseOutsideTestThread) {
+  EXPECT_FALSE(Thread::MainThread::isMainThreadActive());
+  {
+    Event::TestRealTimeSystem real_time_system;
+    DefaultListenerHooks default_listener_hooks;
+    ProdComponentFactory prod_component_factory;
+    const char* argv[] = {"envoy-static",       "--mode", "validate", "--config-path",
```

Comment:
remove whitespace before `"--mode"`

Reply:
Well, clang-format really wants those spaces in there so the `"--mode"`` lines up over the `nullptr`.